Week-3/CSS-ADVANCED/q1.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Question and Answer</title>
+    <style>
+
+        body {
+         font-family: Calibri, sans-serif;
+         font-size: 3em;
+      }
+
+      #answer {
+         color: white;
+         width: 100px;
+         animation-name: moveFraction;
+         animation-duration: 2s;
+         animation-delay: 1s;
+         animation-fill-mode: forwards;
+
+      }
+
+      @keyframes moveFraction {
```

Comment:
It's not clear why the initial position of the` #answer` is set so far to the left` (-355px)`. If it's for animation purposes, consider adding a **comment** to explain.